docs/developer-guide.md

```diff
@@ -39,6 +39,38 @@ In general we perfer **Squash and merge** against `v3` or feature branches. When
 
     **At this stage, changes to ideal output, config, error message and line number are not considered breaking*
 
+## Triage Process
+
+We are happy to accept small fixes and small enhancements through pull requests directly.
+For proposals or large changes, we use the following process to pickup, review and approve issues that aligns with our [roadmap](roadmap.md) and priority:
+
+1. Create an issue on GitHub to start a discussion of the proposal.
+
+2. We assign a team member when an issue is picked up in review meetings.
```

Comment:
>picked up in review meetings. [](start = 44, length = 29)

1. can someone self-assign the issue to himself/herself?
2. do we need some other label to denote that the issue is in picked-up status?